install/docker-compose/environment.rst

```diff
@@ -72,12 +72,24 @@ ELASTICSEARCH_SSL_VERIFY: ``true``
 Memcached
 ---------
 
-MEMCACHED_HOST: ``zammad-memcached``
+MEMCACHE_SERVERS: ``zammad-memcached``
    Provide your own Memcached instance if you already have one existing.
 
-MEMCACHED_PORT: ``11211``
+   .. warning:: Was ``MEMCACHED_HOST`` before 5.0.x!
+
+MEMCACHE_PORT: ``11211``
    Memcacheds default port.
```

Comment:
Sorry, I may not have been clear enough on this. These lines should be removed, that variable does not exist.

Reply:
Ugh dunno where I picked that up sorry.